entities/license/directoryPackage.json

```diff
@@ -7,48 +7,78 @@
       },
       "directory_uid":{
         "type":"string",
-        "description": "The license directory unique identifier. Will determine which directory can sell this package. Null means Available to all directories"
+        "description":"Unique identifier for the license directory. Defines which directory is authorized to sell this package. If null, the package is available to all directories. Once assigned, it cannot be modified."
       },
       "product_unique_code": {
         "type": "string",
-        "description": "The related underlying product unique identifier (app code name, package code...)"
+        "description": "Unique code for the associated underlying product (e.g., app code name, package code)"
       },
       "type": {
         "type": "string",
-        "description": "The type of the directory package. It can be ״plan״ - for the business's main subscription; ״app״ - for a marketplace app subscription; \n ״one_time_purchase״ - for consumables; ״staff_seat״ - for additional seats for the business"
+        "description": "Category of the directory package. Possible values include: ״plan״ for the main business subscription; ״app״ for marketplace app subscriptions; ״SMS״ for text message packages; ״staff_seat״ for additional business seats."
       },
       "display_name": {
         "type": "string",
         "description": "The product's display name"
       },
-      "active":{
-        "type":"boolean",
-        "description": "Indicates that new subscriptions can be created only when this directory package is active",
-        "default":true
+      "quantity": {
+        "type": "number",
+        "description": "The number of items included in the package",
+        "default":1
+      },
+      "payment_type": {
+        "type": "string",
+        "description": "Billing frequency, such as monthly, annual, free, or single charge"
+      },
+      "status":{
+        "type":"string",
+        "description": "Current status of the directory package (e.g., active, inactive)",
+        "default":"active"
       },
       "prices":{
         "type": "array",
         "items":{
           "type": "object",
-          "$ref": "https://vcita.github.io/developers-hub/entities/license/directoryPackage.json"
+          "properties":{
+            "price": {
+              "type": "number",
+              "description": "Cost of the directory package"
+            },
+            "currency": {
+              "type": "string",
+              "description": "Currency for the price (e.g., USD, EUR, GBP)"
+            }
+          }
+        }
+      },
+      "trial":{
+        "type":"number",
+        "description":"Number of days for the trial period. If not specified or set to 0, it indicates no trial period is offered."
+      },
+      "bundles":{
+        "type": "array",
+        "items": {
+          "$ref": "https://vcita.github.io/developers-hub/entities/license/directoryPackageBundle.json"
         }
```

Comment:
I would not add this property. Its not part of the entity. We update the bundles separately and not through this entity. Its not like pricing